Falcon BMS Alternative Launcher/AppRegInfo.cs

```diff
@@ -213,62 +205,52 @@ public void Init(MainWindow mainWindow, string version)
                     overRideSetting = new OverrideSettingFor437(this.mainWindow, this);
                     starter         = new Starter437Internal(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.36 (Internal)":
                     bms_Version     = BMS_Version.BMS436I;
-                    keyFileName     = CommonConstants.DEFAULTKEY + ".key";
                     overRideSetting = new OverrideSettingFor437(this.mainWindow, this);
                     starter         = new Starter436Internal(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.37":
                     bms_Version     = BMS_Version.BMS437;
-                    keyFileName     = CommonConstants.DEFAULTKEY + ".key";
                     overRideSetting = new OverrideSettingFor437(this.mainWindow, this);
                     starter         = new Starter437(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.36":
                     bms_Version     = BMS_Version.BMS436;
-                    keyFileName     = CommonConstants.DEFAULTKEY + ".key";
                     overRideSetting = new OverrideSettingFor436(this.mainWindow, this);
                     starter         = new Starter436(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.35":
                     bms_Version     = BMS_Version.BMS435;
-                    keyFileName = CommonConstants.DEFAULTKEY + ".key";
                     overRideSetting = new OverrideSettingFor435(this.mainWindow, this);
                     starter         = new Starter435(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.34":
                     bms_Version     = BMS_Version.BMS434U1;
-                    keyFileName     = CommonConstants.DEFAULTKEY + ".key";
                     overRideSetting = new OverrideSettingFor434U1(this.mainWindow, this);
                     starter         = new Starter434(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.33 U1":
                     bms_Version     = BMS_Version.BMS433U1;
-                    keyFileName     = CommonConstants.DEFAULTKEY + ".key";
                     overRideSetting = new OverrideSettingFor433(this.mainWindow, this);
                     starter         = new Starter433(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.33":
                     bms_Version     = BMS_Version.BMS433;
-                    keyFileName     = CommonConstants.DEFAULTKEY + ".key";
                     overRideSetting = new OverrideSettingFor433(this.mainWindow, this);
                     starter         = new Starter433(this, this.mainWindow);
                     break;
                 case "Falcon BMS 4.32":
                     bms_Version     = BMS_Version.BMS432;
-                    keyFileName     = "BMS.key";
```

Comment:
BMS 4.32 had a different default key file name so I wonder if we remove it does AL can support 4.32...........who plays 4.32 in 2023 is another question though :)